### ISSUE TITLE
Multi-line preview params being rendered in the preview source

### ISSUE THREAD

**Issue by kcsapko-clio** [allmarkedup/lookbook#141]
Hey all, just want to call out first and foremost that we're loving Lookbook! 

I'd like to follow the format of rendering previews as such in the ViewComponent preview file. What's great about this method is that the preview itself doesn't need to be pulled out into a separate `.html.erb` file, which is especially helpful to reducing the number of files you have when you have multiple variations of previews to show. 

```ruby
def default(theme: :primary)
  render Elements::ButtonComponent.new(theme: theme) do
    'Click me'
  end
end
```

The problem on our end, however, is that we have quite a few params for our ViewComponents. Take this example below. As a result, we tend to format our code so that each param is on a separate line, as such:

```ruby
def default(
  content: "Link",
  href: "https://www.google.com",
  target: nil,
  prefix_icon: nil,
  suffix_icon: nil,
  prefix_icon_aria_label: nil,
  suffix_icon_aria_label: nil
)
  render LinkComponent.new(
    content: content,
    href: href,
    target: target,
    prefix_icon: prefix_icon,
    suffix_icon: suffix_icon,
    prefix_icon_aria_label: prefix_icon_aria_label,
    suffix_icon_aria_label: suffix_icon_aria_label
  ) do
    content
  end
end
```

When we do this, Lookbook actually seems to render everything below the first line (`def default(`) in the preview Source tab itself, as opposed to just rendering whatever is in the actual method itself. As you can imagine, this would be super confusing if someone relied on the Source tab for copy/pasting component code. In reality, I'd imagine we only want lines 9-19 in the screenshot below rendered in the Source tab?

<kbd>![multiline](https://user-images.githubusercontent.com/62913335/189129801-a4b9ae76-9bee-44c1-bed3-282897947f62.png)</kbd>

**Comment by allmarkedup** [comment]
Hi @kcsapko-clio, glad you are liking it :-)

So currently Lookbook tries to remove the method call 'wrapper' from the source by just removing the top and bottom lines from the entire method call, but that clearly won't work in cases like this. It's obviously a perfectly legitimate way to arrange params but I just hadn't considered it! So I'll need to make the logic there a little bit smarter.

I'll label it as a bug because really this should be the expected behaviour and I'll get to it as soon as I can. Appreciate you bringing it to my attention 👍

**Comment by allmarkedup** [comment]
Hey @kcsapko-clio - just to let you know that this has now been fixed on `main` and the fix will go out in the next release, probably later today.

**Comment by kcsapko-clio** [comment]
Legend! Much appreciated @allmarkedup! We're very much looking forward to cleaning up a ton of code on our end with this release 😄

**Comment by allmarkedup** [comment]
@kcsapko-clio Great! I've just released v1.0.8 so I'm going to go ahead and close this now. If you still have any problems after updating feel free to re-open :-)
